src/_data/catalog/widgets.json

```diff
@@ -1629,6 +1629,17 @@
     "link": "https://docs.flutter.io/flutter/widgets/AnimatedWidgetBaseState-class.html",
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "Expanded",
+    "categories": [
+    ],
+    "subcategories": [
+      "Layout helpers"
+    ],
+    "description": "A widget that expands a child of a Row, Column, or Flex..",
+    "link": "https://docs.flutter.io/flutter/widgets/Expanded-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
```

Comment:
needs an icon

Reply:
Are you saying that there is an image that exists already that I can use? (Or is it a general remark that we should be avoiding the use of the placeholder image as much as possible?)